src/models/project.js

```diff
@@ -0,0 +1,58 @@
+const mongoose = require('mongoose');
+
+const applicationSchema = new mongoose.Schema(
+  {
+    applicant: {
+      type: String,
+      ref: 'User',
+      required: true,
+    },
+    date: {
+      type: Date,
+      required: false,
+    },
+    status: {
+      type: String,
+      enum: ['pending', 'accepted', 'rejected'],
+      default: 'pending',
+    },
+  },
+  { timestamps: true }
+);
+
+const projectSchema = new mongoose.Schema(
+  {
+    creator: {
+      type: String,
+      ref: 'User',
+      required: true,
+    },
+    title: {
+      type: String,
+      required: true,
+    },
+    type: {
+      type: String,
+      required: true,
+    },
+    location: {
+      type: String,
+      required: true,
+    },
+    isOpen: {
+      type: Boolean,
+      required: true,
+    },
+    necessarySkills: {
+      type: [String],
+    },
+    date: {
+      type: Date,
+      required: true,
+    },
+    applications: [applicationSchema],
+  },
+  { timestamps: true }
+);
+
+module.exports = mongoose.model('Project', projectSchema);
```

Comment:
This file should not appear in file changes. Could you check what causes this to appear here.